os/fs/driver/mtd/smart.c

```diff
@@ -2163,18 +2163,18 @@ static int smart_scan(FAR struct smart_struct_s *dev)
 #endif
 			}
 
-			fdbg("Duplicate Sector winner=%d, loser=%d\n", winner, loser);
+			fdbg("Duplicate Sector active_sector=%d, inactive_sector=%d\n", winner, loser);
```

Comment:
You can implement these changes in the first commit itself and re push please.
We don't need to add a second commit here.